packages/command/command.go

```diff
@@ -0,0 +1,56 @@
+package command
+
+import (
+	i3 "github.com/denbeigh2000/goi3bar"
+	"os/exec"
+	"bytes"
+	"fmt"
+	"log"
+	"strings"
+)
+
+type CommandItem struct {
+}
+
+type CommandGenerator struct {
+	Label    string `json:"label"`
+	Instance string `json:"instance"`
+	Command  string `json:"command"`
+	Color    string `json:"color"`
+
+	// Identifier for receiving click events
+	Name     string
+}
+
+func (g CommandGenerator) Generate() ([]i3.Output, error) {
+	items := make([]i3.Output, 1)
+
+	items[0].Name = g.Name
+	cmd := exec.Command(g.Command)
+	if (len(g.Instance) > 0) {
+		cmd.Env = []string{fmt.Sprintf("BLOCK_INSTANCE=%s", g.Instance)}
+	}
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		log.Panicf("Failed to execute %s: %v", g.Command, err)
```

Comment:
Don't call `log.Panicf` here - the bar will crash entirely. Use `log.Errorf` instead.